src/components/SearchBox/SearchBox.scss

```diff
@@ -0,0 +1,50 @@
+.searchbox {
+    $input-font-size: 30px;
+    $input-line-height: 70px;
+    $font-family: Roboto Slab, sans-serif;
+
+    position: relative;
+    width: 500px;
+    margin: 50px auto;
+
+    input {
+        height: 60px;
+        width: 100%;
+        min-width: 100%;
+        padding: 0;
+        border-radius: 0;
+        line-height: $input-line-height;
+        background-color: transparent;
+        color: #757575;
+        font-size: $input-font-size;
+        border: none;
+        outline: none;
+        border-bottom: 3px solid #757575;
+        font-family: $font-family;
+
+        &:focus {
+            + .input-highlight {
+                border-top: 3px solid #8162FF;
+            }
+        }
+    }
+
+    ::placeholder {
+        color: #999;
+    }
+
+    .input-highlight {
+        font-size: $input-font-size;
+        user-select: none;
+        line-height: $input-line-height;
+        border-top: 3px solid white;
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        max-width: 100%;
+        height: 0;
+        color: transparent;
+        font-family: $font-family;
+        overflow: hidden;
+    }
+}
```

Comment:
Incorrect line ending

Reply:
Fixed. @sukritchhabra need to fix the linter for scss files.